setup.cfg

```
@@ -62,6 +62,8 @@ xfail_strict=True
 filterwarnings=
     # Turn warnings that aren't filtered into exceptions
     error
+    # https://bugs.python.org/issue45097
```

So we can remember to remove in the future.

```suggestion
    # Workaround for Python 3.9.7 (see https://bugs.python.org/issue45097)
```